### PR TITLE
Fifo animal shelter

### PR DESCRIPTION
**Pull request by ZenAtomsk**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/fifoAnimalShelter/fifo-animal-shelter.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Node = require('../node.js');
+
+class AnimalQueue {
+  constructor() {
+    this.storage = nul
+    l;
```
Comment (adrianhuebner) [inline javascript/code-challenges/fifoAnimalShelter/fifo-animal-shelter.js:8]: This is causing issues because you accidentally spaced the last l on a separate line

File: javascript/code-challenges/fifoAnimalShelter/fifo-animal-shelter.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Node = require('../node.js');
```
Comment (adrianhuebner) [inline javascript/code-challenges/fifoAnimalShelter/fifo-animal-shelter.js:3]: This needs to be `const Node = require('../linked-list/node');`
